### PR TITLE
Feature/recycler view for notes

### PR DESCRIPTION
**Pull request by theo-damiani**
PR so that the note selection uses a recycler view.
No view model yet as the note-database PR #83 has not been merged. But this will not add a lot of work. The bigger part (recycler view and Holder is done).

### CODE REVIEW

File: app/src/main/java/com/github/steroidteam/todolist/view/adapter/NoteAdapter.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    @Override
+    public int getItemCount() {
+        return noteList.size();
+    }
+
+    public UUID getIdOfNote(int index) {
+        return noteList.get(index).getId();
+    }
+
+    public interface NoteCustomListener {
```
Comment (sydneyhauke) [inline app/src/main/java/com/github/steroidteam/todolist/view/adapter/NoteAdapter.java:50]: Is there no other way than creating its own custom listener ?

**Review by sydneyhauke — APPROVED** [review]

File: app/src/main/java/com/github/steroidteam/todolist/view/adapter/NoteAdapter.java (hunk 1 above)
Comment (theo-damiani) [reply app/src/main/java/com/github/steroidteam/todolist/view/adapter/NoteAdapter.java:50]: I think but this way I can add other methods in the future and it was easier to recover in the activity the Holder of a note.
But sure if you prefer I can maybe look to change this.

**Review by shoutizix — APPROVED** [review]
Nice !

This works perfectly on my phone ! The cardview for the notes is to display the images even in the selection am I right ?

I think this would be great to merge it pretty fast, because everytime we'll make modification on the notes part we'll need to adapt this PR to not break code, so the sooner the better I think.

**Comment by theo-damiani** [comment]
> 
> 
> Nice !
> 
> This works perfectly on my phone ! The cardview for the notes is to display the images even in the selection am I right ?
> 
> I think this would be great to merge it pretty fast, because everytime we'll make modification on the notes part we'll need to adapt this PR to not break code, so the sooner the better I think.

Thanks!
This the idea but as there isn't the viewmodel I just put the logo in every notes.

**Pull request merged**